### PR TITLE
Fuzz io standards

### PR DESCRIPTION
**Pull request by yrabbit**
Add IO standards fuzzing (single end LVTTL33, LVCMOS33, LVCMOS25, LVCMOS18, LVCMOS15, LVCMOS12, SSTL25_I, SSTL25_II, SSTL33_I, SSTL33_II, SSTL18_I, SSTL18_II, SSTL15, HSTL18_I, HSTL18_II, HSTL15_I). 
Add DRIVE attribute support.
Use io standards during packing.
Tangnano example.

Absent: 
- check for permissible attribute combinations
- default standard detection (probably for all boards it is LVCMOS18)
- all differential modes
- not compared to bitstream after vendor packer
- something else:)

### CODE REVIEW

**Comment by pepijndevos** [comment]
Seems like on GW1N-4 there are some warnings that cause it to fail
```
 Warning (CT1108) : "/tmp/tmp5w6cog09/top.cst" #line(2) | Illegal port attribute value specified 'DRIVE = 12' on 'inst4957_IOB_IOBUF'.
Warning (CT1108) : "/tmp/tmp6f4xrssd/top.cst" #line(3) | Illegal port attribute value specified 'DRIVE = 4' on 'inst4125_IOB_IOBUF'.
Warning (CT1108) : "/tmp/tmp6f4xrssd/top.cst" #line(4) | Illegal port attribute value specified 'DRIVE = 4' on 'inst4225_IOB_IOBUF'.
```

**Comment by yrabbit** [comment]
These are normal warnings, there are a lot of them in logs of other boards, they are used to try permissible attribute values. I don't understand what the GW1N-4 is guilty of yet...

make: *** [Makefile:13: GW1N-4_stage1.pickle] Killed

**Comment by yrabbit** [comment]
Reduce the amount of memory used by refusing to enumerate intentionally incongruent attribute values (according to the documentation). Switch to a ``lazy'' version of the map.

**Comment by pepijndevos** [comment]
Looks good! Will review more thoroughly later.

File: examples/Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 YOSYS ?= yosys
 NEXTPNR ?= nextpnr-gowin
 
+all: attosoc-tec0117.fs nanolcd-tangnano.fs blinky-tec0117.fs blinky-runber.fs \
+	blinky-tangnano.fs blinky-honeycomb.fs shift-tec0117.fs shift-runber.fs \
+	shift-tangnano.fs shift-honeycomb.fs iob-tangnano.fs iob-default-flags-tangnano.fs
 .PHONY: all clean
 
 all: attosoc-tec0117.fs nanolcd-tangnano.fs blinky-tec0117.fs blinky-runber.fs blinky-tangnano.fs blinky-honeycomb.fs shift-tec0117.fs shift-runber.fs shift-tangnano.fs shift-honeycomb.fs
```
Comment (pepijndevos) [inline examples/Makefile:9]: wait now there are two `all` targets? Merge error?

**Pull request merged**